gix-path/tests/path/env.rs

```diff
@@ -26,6 +24,14 @@ fn installation_config() {
     );
 }
 
+#[test]
+fn core_dir() {
+    assert!(
+        gix_path::env::core_dir().expect("Git is always in PATH").is_dir(),
```

Comment:
I did not immediately realize what the `expect` message was expressing. Assuming this understanding of mine is correct, it might be a bit clearer, but this is subjective and I am not sure:

```suggestion
        gix_path::env::core_dir().expect("Git is always in PATH when we run tests").is_dir(),
```

Reply:
There I was very fast and sloppy and think the patch helps to be a bit clearer, without claiming Git is installed on each machine in the world :D.